src/test/ui/const-generics/defaults/type-after-const-requires-default.rs

```diff
@@ -0,0 +1,10 @@
+// run-pass
```

Comment:
afaik the name is somewhat misleading here?

I think we probably should move this out of `defaults`, considering that it doesn't use defaults